src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Models/Cohort/AddDraftApprenticeshipValidationTests.cs

```diff
@@ -137,6 +134,62 @@ public void StartDate_CheckNotBeforeMay2017_Validation(DateTime? startDate, bool
             Assert.AreEqual(expectedErrorMessage, startDateError.ErrorMessage);
         }
 
+        [TestCase(null, true)]
+        [TestCase("2017-04-30", false)]
+        [TestCase("2017-05-01", true)]
+        public void ActualStartDate_CheckNotBeforeMay2017_Validation(DateTime? startDate, bool passes)
+        {
+            var utcStartDate = startDate.HasValue
+                ? DateTime.SpecifyKind(startDate.Value, DateTimeKind.Utc)
+                : default(DateTime?);
+
+            _fixture.WithCurrentDate(new DateTime(2017, 5, 1))
+                .AssertValidationForProperty(() => _fixture.DraftApprenticeshipDetails.ActualStartDate = utcStartDate,
+                    nameof(_fixture.DraftApprenticeshipDetails.ActualStartDate)
+                    , passes);
+        }
+
+        [TestCase(null, "2019-01-01", "2019-12-31", true, Description = "Start date not specified")]
```

Comment:
similar to the other PR should this not be false in this scenario? Again probably my misunderstanding but worth challenging.